firmware/firmware.ino

```diff
@@ -214,6 +215,42 @@ void handleCommand(const char *buffer)
       }
       entry_object = platypus::sensors[sensor_idx];
     }
+    //check for compensation commands 'c_'
+    /*else if (entry_name[0] == 'c'){
+      Serial.println("compensation data detected");
+      //ec compensation data
+      if (entry_name[2] == 'e'){
+        Serial.println("Got ec data, looking for sensors...");
+        //loop through sensors looking for sensors that care about ec (DO)
+        for (int i=0; i < 4; i++){
+          if (strncmp(platypus::sensors[i]->name(), "atlas_do", 7) == 0){
+            Serial.print("sensor ");
+            Serial.print(i);
+            Serial.println(" cares about ec");
+            entry_object = platypus::sensors[i];
+          }
+        }
+      } else if (entry_name[2] == 't'){
+        Serial.println("Got temp data, looking for sensors...");
+        for (int i=0; i < 4; i++){
+          Serial.println(platypus::sensors[i]->name());
+          if (strncmp(platypus::sensors[i]->name(), "atlas", 5) == 0){
+            Serial.print("sensor ");
+            Serial.print(i);
+            Serial.println(" cares about temp");
+            entry_object = platypus::sensors[i];
+          }
+        }
+      } else {
+        reportError("Unknown compensation entry.", buffer);
+        return;
+      }
+
+      //No sensors to compensate
+      if (entry_object == NULL){
+        return;
+      }
+    }*/
```

Comment:
For Sensor compensation. I'll remove it from master for now, then add it back in when I start the Feature-SensorCompensation branch